fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/RepositoryObject.java

```diff
@@ -55,11 +55,13 @@
     protected PremisLogger premisLog;
 
     protected RepositoryObject(PID pid, RepositoryObjectLoader repositoryObjectLoader,
-            RepositoryObjectDataLoader dataLoader, RepositoryObjectFactory repoObjFactory) {
+            RepositoryObjectDataLoader dataLoader, RepositoryObjectFactory repoObjFactory,
```

Comment:
what was the reason for adding a pid minter to RepositoryObject and each of its subclasses? i thought at one point we were thinking of restricting its use mainly to the factory